aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -81,6 +82,19 @@ pub fn allow_module_bundle_for_test() {
     MODULE_BUNDLE_DISALLOWED.store(false, Ordering::Relaxed);
 }
 
+/// Contains information about execution failure.
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
this looks a bit unorthodox ... maybe consider thiserror?

Reply:
okay, went and read the code more... you're introducing some interesting Move <-> Rust code here... this feels like it belong as a proper type... can't this also be used to deserialize the output? maybe it belongs in types?

Reply:
What do you mean deserialize the output? I moved it to types